public/shared/channels.js

```diff
@@ -47,4 +47,5 @@ module.exports = {
   SET_SCHEMA_CHANNEL: 'schema:set',
   DELETE_SCHEMA_CHANNEL: 'schema:delete',
   OPEN_URL_IN_BROWSER_CHANNEL: 'open:browser',
+  GET_PIPELINE_CHANNEL: 'pipeline:get',
```

Comment:
I think you missed `s` for `PIPELINES` in most of you nomenclature. You will need two calls to get all pipelines vs get one pipeline. In this case we are dealing with `GET_PIPELINES_CHANNEL`.